Classes/Utility/ImageVariantsUtility.php

```diff
@@ -29,22 +30,27 @@ class ImageVariantsUtility
     protected static $defaultVariants = [
         'default' => [
             'breakpoint' => 1200,
-            'width' => 1100
+            'width' => 1100,
+            'widthRetina' => 2200
```

Comment:
Please remove default variants, we only need to enable the user to configure retina images.
I would not like to deliver them by default.